spec/nylas/message_spec.rb

```diff
@@ -186,6 +186,23 @@
     end
   end
 
+  describe "update_folder" do
+    it "move message to new `folder`" do
+      folder_id = "9999"
+      api = instance_double(Nylas::API, execute: "{}")
+      message = described_class.from_json('{ "id": "message-1234" }', api: api)
+      allow(api).to receive(:execute)
+
+      message.update_folder(folder_id)
+
+      expect(api).to have_received(:execute).with(
+        method: :put,
+        path: "/messages/message-1234",
+        payload: { folder_id: "9999" }.to_json
```

Comment:
Here you could refer to the variable `folder_id` instead of the integer literal again - may make it less brittle.

Reply:
Yes we can. Fixed here 86e2a1fc6bb1a76b816c63819b05ee6cf707d325